### ISSUE TITLE
Logging to Logstash via HTTPS uses plain HTTP connection

### ISSUE THREAD

**Issue by kefiras** [ansible/awx#2048]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - UI

##### SUMMARY
Logs are being sent in plain HTTP when "LOG_AGGREGATOR_TYPE": "logstash" and "LOG_AGGREGATOR_PROTOCOL": "https" is chosen.

##### ENVIRONMENT
* AWX version: 1.0.4.0
* AWX install method: docker on linux,
* Ansible version:  2.5.4
* Web Browser: Chrome

##### STEPS TO REPRODUCE

Configure settings/logging to:
`
"LOG_AGGREGATOR_TYPE": "logstash"
"LOG_AGGREGATOR_PROTOCOL": "https"
`
 hit Test to send test logs to external aggregator

##### EXPECTED RESULTS

Data encrypted in TLS 

##### ACTUAL RESULTS

Data sent in plain HTTP

##### ADDITIONAL INFORMATION

tcpdump pcap file attached
<img width="707" alt="awx-ext-agg-log" src="https://user-images.githubusercontent.com/15342111/42371526-0813a960-8110-11e8-83e0-05b1989d6fff.png">

**Comment by ryanpetrello** [comment]
@kefiras I'm having trouble reproducing this behavior.  Can you share your settings at:

```
HTTP GET https://tower.example.org/api/v2/settings/logging/
```

**Comment by ryanpetrello** [comment]
Actually, I think I see what's up.  I'm digging...

**Comment by ryanpetrello** [comment]
@kefiras what's going on here is that the `Protocol: HTTPS` option _actually_ supports HTTP *and* HTTPS, depending on how you specify the hostname.  The issue is that if you _don't_ specify the protocol in the hostname field (e.g., `example.awx.org`), it assumes _http_, not _https_ (which I'd argue is a bad/confusing fallback behavior).  To get around this for now, you should be able to change the aggregator hostname field to e.g., `https://example.awx.org`).  I'm currently working on a longer-term fix for this bug.

**Comment by kefiras** [comment]
@ryanpetrello Ok thanks for update. For the time being documentation upgrade should be enough I assume :)

**Comment by ryanpetrello** [comment]
@kefiras we've just released 1.0.7, which we believe resolves the underlying issue here. You can try out here: https://github.com/ansible/awx/releases/tag/1.0.7

Let us know if you're still seeing this issue after installing the latest awx - thanks!
